### PR TITLE
remove messages maintainer message

### PR DESCRIPTION
**Pull request by nhooyr**
I don't think anyone finds this message very useful.

### CODE REVIEW

**Comment by ZyX-I** [comment]
I also do not think that Bram could be message maintainer in Neovim.

**Comment by ZyX-I** [comment]
It is bad: `:messages` is not tested good enough for tests to fail because of this change (they fail because of unused variable warning).

**Comment by justinmk** [comment]
I think Bram added some tests for messages module, recently.

**Comment by tweekmonster** [comment]
Maybe instead of removing it, it should be replaced with a "tip of the day" or the only tip that new users need: `Type :q to quit`.

File: src/nvim/message.c [hunk 1]
```diff
@@ -706,12 +706,6 @@ void ex_messages(exarg_T *eap)
 
   msg_hist_off = TRUE;
 
-  s = os_getenv("LANG");
```
Comment (justinmk) [inline src/nvim/message.c]: the declaration of `s` on line 705 should be deleted.

Comment (nhooyr) [inline src/nvim/message.c]: fixed

**Comment by justinmk** [comment]
All of the `.po` items related to this must be deleted, as well. Use `git grep 'Messages maintainer'` to find all lines that must be changed.

Also, this explains the purpose of this message:

https://github.com/vim/vim/blob/580164481924ed8611eb79f0247a0eb1ca0b3b9a/src/po/README.txt#L82-L85

though I agree it is noise in `:messages`. And no one actually followed this advice except maintainer of `en_GB.po`.

**Comment by nhooyr** [comment]
Actually, there are many maintainers in there that did follow it. But I removed everything. You couldn't see it with grep because they are in different languages. For example [here](https://github.com/neovim/neovim/blob/d622e9c41635d9c0a65b8725e49ce476838d1aa2/src/nvim/po/af.po#L4310).

**Comment by justinmk** [comment]
Yes, my mistake. The maintainers are listed at the tops of the files, so still ok to remove those items.

**Pull request merged**